build/actions/yarn.lock

```diff
@@ -3,60 +3,71 @@
 
 
 "@actions/core@^1.2.6":
-  version "1.9.1"
-  resolved "https://registry.yarnpkg.com/@actions/core/-/core-1.9.1.tgz#97c0201b1f9856df4f7c3a375cdcdb0c2a2f750b"
-  integrity sha512-5ad+U2YGrmmiw6du20AQW5XuWo7UKN2052FjSV7MX+Wfjf8sCqcsZe62NfgHys4QI4/Y+vQvLKYL8jWtA1ZBTA==
+  version "1.10.1"
```

Comment:
There's a lot of other changes in these files - are these all really from updating follow-redirects?

Reply:
Those seem to be causing failures too - so if you really want to just update follow-redirects then you should start over and manually update the yarn.lock files to target the new version for that package only

Reply:
Should we be updating these versions as an override in the relevant package.json files instead?

Reply:
I deleted these yarn.lock files and regenerated them so that the resolved version of follow-redirects, but a bunch of other unrelated packages also got updated, so I don't think this was the right approach :/ 

This package is an indirect dependency from axios, so would using resolutions be a better way to force the version to 1.15.4?

Reply:
Are we able to bump the packages that depend on follow-redirects instead? I'm looking through the history of some resolutions changes, and seems like it can cause breaking changes for other packages.

Reply:
Yeah, resolutions should only be a very last resort since it can cause problems when updating dependencies later. 

Kim and I chatted directly and she sent out another PR, and we'll work on updating the guide detailing how to update dependencies in the future.